python/packages/zarathustra/skills/subgraph_query_abci/rounds.py

```diff
@@ -97,18 +120,14 @@ def end_block(self) -> Optional[Tuple[BaseSynchronizedData, Enum]]:
         return synchronized_data, Event.DONE
 
 
-
-class LoadSubgraphComponentsRound(CollectionRound):
+class LoadSubgraphComponentsRound(CollectSameUntilThresholdRound):
     """LoadSubgraphComponentsRound"""
 
     payload_class = LoadSubgraphComponentsPayload
-    payload_attribute = ""  # TODO: update
     synchronized_data_class = SynchronizedData
-
-    def end_block(self) -> Optional[Tuple[BaseSynchronizedData, Enum]]:
-        """Process the end of the block."""
-        synchronized_data = self.synchronized_data
-        return synchronized_data, Event.DONE
+    done_event = Event.DONE
+    collection_key = "subgraph_config"
+    selection_key = get_name(SynchronizedData.most_voted_subgraph_config)
```

Comment:
as there is no `NO_MAJORITY` event yet, the new `AbciApp`s need to be updated to include this.

Reply:
noted